docs/man/man1/hal_parport.1

```diff
@@ -90,17 +90,17 @@ The following lists the input and output pins by the type setting used in the cf
 Reads physical input pins of all ports and updates HAL \-in and \-in\-not pins.
 .TP
 \fBparport.<p>.write (funct)
-Reads HAL \-out pins of port <p> and updates that port’s physical output pins.
+Reads HAL \-out pins of port <p> and updates that port's physical output pins.
 .TP
 \fBparport.write\-all (funct)
 Reads HAL \-out pins of all ports and updates all physical output pins.
 .TP
 \fBparport.<p>.reset (funct)
-Waits until \fIreset\-time \fRhas elapsed since the associated write, then resets pins to values indicated by \fI\-out\-reset \fRand \fI\-out\-invert \fRsettings. reset must be later in the same thread as write. 'If '\fI\-out\-reset \fRis TRUE, then the reset function will set the pin to the value of \fI\-out\-invert\fR. This can be used in conjunction with stepgen’s doublefreq to produce one step per period. The stepgen stepspace for that pin must be set to 0 to enable doublefreq.
+Waits until \fIreset\-time \fRhas elapsed since the associated write, then resets pins to values indicated by \fI\-out\-reset \fRand \fI\-out\-invert \fRsettings. reset must be later in the same thread as write. 'If '\fI\-out\-reset \fRis TRUE, then the reset function will set the pin to the value of \fI\-out\-invert\fR. This can be used in conjunction with stepgen's doublefreq to produce one step per period. The stepgen stepspace for that pin must be set to 0 to enable doublefreq.
```

Comment:
I think here we should use U+2019 [[1](https://en.wikipedia.org/wiki/Apostrophe#Unicode)] or U+02BC [[2](https://tedclancy.wordpress.com/2015/06/03/which-unicode-character-should-represent-the-english-apostrophe-and-why-the-unicode-committee-is-very-wrong/)]